modules/ztp-telco-ran-software-versions.adoc

```diff
@@ -52,6 +52,9 @@ The Red Hat telco RAN DU {product-version} solution has been validated using the
 |{rh-rhacm-first}
 |2.11
 
+|{mce}
+|2.11
+
```

Comment:
I think the intent of this bug was to update the table in step 2a under "Configuring the hub cluster with ArgoCD" here: https://docs.openshift.com/container-platform/4.16/edge_computing/ztp-preparing-the-hub-cluster.html#ztp-configuring-hub-cluster-with-argocd_ztp-preparing-the-hub-cluster
Thanks!

Reply:
Oh sorry! Do we need to have add OCP 4.17 as well then?